lib/studies/reply/mail_view_page.dart

```diff
@@ -19,24 +19,24 @@ class MailViewPage extends StatelessWidget {
       body: SafeArea(
         bottom: false,
         child: Container(
-          color: Theme.of(context).cardColor,
-          child: Padding(
-            padding: const EdgeInsetsDirectional.only(
-              top: 42,
-              start: 20,
-              end: 20,
-            ),
-            child: Column(
-              crossAxisAlignment: CrossAxisAlignment.start,
-              children: [
-                _MailViewHeader(
-                  email: email,
-                ),
-                const SizedBox(height: 32),
-                Expanded(
-                  child: _MailViewBody(message: email.message),
-                ),
-              ],
+          height: double.infinity,
+          child: Material(
+            color: Theme.of(context).cardColor,
```

Comment:
In general we try to use values from the colorscheme. Would `Theme.of(context).colorScheme.surface` work here?

Reply:
(apologies because I know this wasn't actually part of your change, only do it if it's no-op!)

Reply:
`surface` wouldn't work in this case, but I will revisit this in another PR along with other color values that can be inherited from colorScheme.